app/backend/middleware/session.ts

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+import expressSession from 'express-session';
+import connectPgSimple from 'connect-pg-simple';
+import { pgPool } from '../db/setup-pg';
+
+const PgSession = connectPgSimple(expressSession);
+const sessionSecret =
+  process.env.SESSION_SECRET || crypto.randomBytes(32).toString();
```

Comment:
While a random default is a good default, it will change every time we boot the app rendering prior sessions invalid and preventing clustering. Let's be absolutely sure to document this in the Readme.